test/mcmc/abstractmcmc.jl

```diff
@@ -112,8 +117,7 @@ end
 
 @testset "External samplers" begin
     @testset "AdvancedHMC.jl" begin
-        # Try a few different AD backends.
-        @testset "adtype=$adtype" for adtype in [AutoForwardDiff(), AutoReverseDiff()]
+        @testset "adtype=$adtype" for adtype in ADUtils.adbackends
```

Comment:
```suggestion
        @testset "adtype=$adtype" for adtype in [AutoForwardDiff(), AutoReverseDiff()]
```
reverting this for now should make the tests pass. @mhauru would you be happy to accept this change for the time being?